src/db/Views/index.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+require_once __DIR__ . './../../../config/bootstrap.php';
+
+use db\Services\HttpService;
+use db\Repositories\ArticleRepository;
+
+$articleRepository = new ArticleRepository($entityManager);
+$httpService = new HttpService();
+
+if (isset($_POST['create'])) {
+    $articleRepository->create([
+        'title' => $_POST['new-article-title'],
+        'description' => $_POST['new-article-description'],
+    ]);
+}
+
+if (isset($_POST['update'])) {
+    $articleRepository->update(
+            $_POST['update-article-id'],
+            [
+                'title' => $_POST['update-article-title'],
+                'description' => $_POST['update-article-description'],
+            ],
+    );
+
+    unset($_GET['update']);
+}
+
+if (isset($_GET['delete'])) {
+    $articleRepository->delete($_GET['delete']);
+
+    unset($_GET['delete']);
+}
+
+$searchCriteria = [];
+
+if (isset($_GET['search'])) {
+    $searchCriteria = ['title' => $_GET['search-title']];
+}
+
+$articles = $articleRepository->index(
```

Comment:
better use Doctrine Paginator
https://www.doctrine-project.org/projects/doctrine-orm/en/2.11/tutorials/pagination.html

Reply:
I used it at first, it even can be viewed in first commit of this pull request(ArticleRepository::indexWithQueryBuilder)